cluster-scope/bundles/openshift-pipelines/kustomization.yaml

```diff
@@ -4,4 +4,5 @@ kind: Kustomization
 resources:
   - ../../base/core/namespaces/openshift-pipelines
   - ../../base/operators.coreos.com/subscriptions/openshift-pipelines-operator-rh
+patchesStrategicMerge:
```

Comment:
why do we need a patch for this?

Reply:
mentioned in the desciption,
it is installed and created by openshift-pipeline operator

Reply:
oh i see, the patch would work on the base and overlays.
we might have to ignore the label somehow tough then.